src/js/components/LoginForm.js

```diff
@@ -90,15 +90,20 @@ export default class LoginForm extends Component {
     const center = ! align || 'stretch' === align || 'center' === align;
 
     const errorsNode = errors.map((error, index) => {
-      let errorComponent;
       if (error) {
-        errorComponent = (
+        let errorMessage;
+        if (React.isValidElement(error)) {
+          errorMessage = error;
+        } else {
+          errorMessage = <FormattedMessage id={error} defaultMessage={error} />;
```

Comment:
That makes sense.  I considered leaving it out and having this be a breaking change, but I figured it was better to be less disruptive.  Thanks for merging it!